phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.css

```diff
@@ -218,11 +218,15 @@
 /*******************************************************/
 /************************ logo *************************/
 /*******************************************************/
-#logoText {
+#logo {
+    -fx-background-image: url('/org/phoenicis/javafx/views/common/phoenicis.png');
+    -fx-background-size: 1.8em;
+    -fx-background-repeat: no-repeat;
+    -fx-background-position: left center;
+    -fx-label-padding: 0 0 0 2.2em;
+    -fx-vpos: center;
     -fx-font-family: "Maven Pro";
     -fx-font-size: 1.8em;
```

Comment:
I think this line is the reason, the image is not scaling correctly.
If I understand #746 correctly, you use the `-fx-font-size` at the root level to define the required scaling.
This means that you're overriding the used scaling for `#logo`, which in my understanding leads to the logo being always the same size.